agent/bench-scripts/pbench-trafficgen

```diff
@@ -1149,9 +1178,9 @@ else
 	done
 fi
 if [ $rc -eq 0 ]; then
-	echo "$script_path/postprocess/generate-benchmark-summary $benchmark_name $orig_cmd $benchmark_run_dir" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
+	echo "$script_path/postprocess/generate-benchmark-summary \"$benchmark_name\" \"$orig_cmd\" \"$benchmark_run_dir\"" >"$benchmark_run_dir/generate-benchmark-summary.cmd"
+	chmod +x "$benchmark_run_dir/generate-benchmark-summary.cmd"
```

Comment:
Why is "$benchmark_run_dir/generate-benchmark-summary.cmd" generated at all? For documentation? Should it be used on line 1183?

Reply:
Seems to be an artifact of how all these benchmark scripts were created.  Added a commit to fix them all.